src/ng/directive/attrs.js

```diff
@@ -181,7 +181,7 @@
       </file>
     </example>
  *
- * @element INPUT
+ * @element INPUT, BUTTON, SELECT
```

Comment:
Theoretically, it can go on any element, but I guess it makes sense to only recommend it for form controls.
What about `textarea`?

Reply:
u're right, there might be some elements missing. At this moment it is just input and i kinda hatet, my idea shows me some errors, so i just added what i need. Due to the build fail, I don't think, that i will improve this pull request, or make any other request. So IF you want to improve the documentation, you could just add the form elements to the element tag and i will be pleased

Reply:
You can still add textarea in a new commit, the build failures are unrelated to your changes